frontend/src/app/auth/components/confirm-email-page/confirm-email-page.component.ts

```diff
@@ -32,6 +32,10 @@ export class ConfirmEmailPageComponent implements OnInit {
         this.notifService.success(
           'Email confirmed. You will be redirected to your profile',
         );
+        this.router.navigate(['../..', 'organizations'], {
+          replaceUrl: true,
+          relativeTo: this.route,
+        });
```

Comment:
Do we also redirect to organizations page after login?

Reply:
Added redirection to organizations page after login